src/Integration.Vsix/SonarLintNotificationsPackage.cs

```diff
@@ -75,6 +76,8 @@ private async System.Threading.Tasks.Task InitAsync()
             logger = await this.GetMefServiceAsync<ILogger>();
             logger.WriteLine(Resources.Strings.Notifications_Initializing);
 
+            sseSessionManager = await this.GetMefServiceAsync<ISSESessionManager>();
```

Comment:
Probably not the right package for this. Not sure where else to put it. Our packages are a bit of a mess.